src/apps/auth/handlers/authorizer.ts

```diff
@@ -75,7 +74,7 @@ const loginCheck = async (userId: string, token: string) => {
     accessToken: token,
     userId: userId,
     expired: {
-      $lte: moment().toDate(),
+      $gte: moment().toDate(),
```

Comment:
why do you use `$gte` here?

Reply:
The code logic is wrong here
https://github.com/quangtran85/serverless-tutorial/pull/8/files#diff-b1580df8d02abbda65771a90a931cd8ca0e67e444f14746fe2fdff4567d0cd56R150
As a result, getting data here would no longer be correct, so I fixed it